docs/Coding-Guidelines.md

```diff
@@ -11,3 +11,4 @@ languages used in implementation or testing.
   - [Markdown](Coding-Guidelines/Languages-Markdown.md)
 - [Experimental](Coding-Guidelines/Experimental.md)-only features
   - Use [TerminalVelocity](Coding-Guidelines/TerminalVelocity.md) tag and disable features
+- [Api Contracts](WinRTAPIContracts.md)-make a stable API.
```

Comment:
`Coding-Guidelines/WinRTAPIContracts`

This list will grow so we're keeping all coding guidelines docs (other than the top level one) in the Coding-Guidelines subdir. To prevent docs\* from becoming a huge mosh pit of many topics.